perf.cmd

```diff
@@ -0,0 +1,10 @@
+@echo off
```

Comment:
This file does not seem to be mentioned/described in the instructions. Is that intended? i.e. Should we use this file to run perf tests? 
Also, if yes, could we rename the file to something more descriprive, e.g. testperfromance.cmd